ansible/sample_production/group_vars/setup_storage_service_class.yml

```diff
@@ -0,0 +1,14 @@
+api_client: "{{ ansible_env.API_CLIENT }}"
```

Comment:
Let's not repeat `api_client` or `priv_key_file` in both setup_storage_service_class.yml and setup_protection_polices.yml. User should only need to specify it once in a common var file (maybe `all`)

Also, let's not name the vars file the same name as the Ansible playbook. This makes it very confusing to refer to the vars file. For vars file, you can use policy.yml like in [python/config](https://github.com/pure-aferrario/fusion-client-setup/tree/main/python/config)